guides/common/modules/proc_creating-a-compliance-policy.adoc

```diff
@@ -1,44 +1,33 @@
 [id="Creating_a_Compliance_Policy_{context}"]
 = Creating a Compliance Policy
 
-With {Project}, you can create a compliance policy to scan your content hosts to ensure that the hosts remain compliant to your security requirements.
-
-You can use either Puppet or Ansible to deploy the compliance policy to your hosts.
-Note that Puppet runs by default every 30 minutes.
-If you assign a new policy, the next Puppet run synchronizes the policy to the host.
-However Ansible does not perform scheduled runs.
-To add a new policy, you must run Ansible role manually or using remote execution.
-For more information about remote execution, see {ManagingHostsDocURL}Configuring_and_Setting_Up_Remote_Jobs_managing-hosts[Configuring and Setting up Remote Jobs] in _{ManagingHostsDocTitle}_.
+Use this procedure to create a compliance policy, which will be used to scan your hosts to ensure that the hosts remain compliant to your security requirements.
 
 .Prerequisites
-
-Before you begin, you must decide whether you want to use a Puppet or Ansible deployment.
-
-* For Puppet deployment, ensure that each host that you want to audit is associated with a Puppet environment.
-For more information, see xref:Importing_OpenSCAP_Puppet_Modules_{context}[].
-* For Ansible deployment, ensure that you import the `theforeman.foreman_scap_client` Ansible role.
-For more information about importing Ansible roles, see {ManagingConfigurationsAnsibleDocURL}Getting_Started_with_Ansible_in_{project-context}_ansible[Getting Started with Ansible in {Project}] in _{ManagingConfigurationsAnsibleDocTitle}_.
+* You have configured {Project} for your selected xref:compliance-policy-deployment-options_{context}[deployment method].
+* You have available SCAP contents, and eventually tailoring files, in {Project}.
```

Comment:
Unsure about the plural of content. Probably applies to multiple instances.

Reply:
I insist. Singular "SCAP content" denotes a single data-stream file. Also, plural is used the WebUI.